lib/jellyfish_web/controllers/room_controller.ex

```diff
@@ -82,10 +82,14 @@ defmodule JellyfishWeb.RoomController do
       |> render("show.json", room: room, jellyfish_address: jellyfish_address)
     else
       {:error, :invalid_max_peers} ->
-        {:error, :bad_request, "maxPeers must be a number"}
+        max_peers = Map.get(params, "maxPeers")
+        {:error, :bad_request, "maxPeers must be a number received maxPeers was: #{max_peers}"}
```

Comment:
Let's write all of those error messages as:

```suggestion
        {:error, :bad_request, "Expected maxPeers to be a number, got: : #{max_peers}"}
```